File: book/packt/Java.EE.6.with.GlassFish.3.Application.Server/0363OS_02_revised_code/simpleapp/src/main/java/net/ensode/glassfishbook/simpleapp/SimpleServlet.java
package net.ensode.glassfishbook.simpleapp;

import java.io.IOException;
import java.io.PrintWriter;

import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

public class SimpleServlet extends HttpServlet
{
  protected void doGet(HttpServletRequest request, HttpServletResponse response)
  {
    try
    {
      response.setContentType("text/html");
      PrintWriter printWriter = response.getWriter();
      printWriter.println("<h2>");
      printWriter
          .println("If you are reading this, your application server is good to go!");
      printWriter.println("</h2>");
    }
    catch (IOException ioException)
    {
      ioException.printStackTrace();
    }
  }
}

File: book/packt/Scala.for.Java.Developers/Chapter 03/SampleProject/src/main/scala/com/samples/SimpleServlet.scala
package com.samples

import scala.xml.NodeSeq
import javax.servlet.http._

class SimpleServlet extends HttpServlet {

  override def doGet(request: HttpServletRequest, response: HttpServletResponse) {

    response.setContentType("text/html")
    response.setCharacterEncoding("UTF-8")

    val responseBody: NodeSeq =
      <html><body><h1>Hello, world!</h1></body></html>

    response.getWriter.write(responseBody.toString)
  }
}